views/partials/shelves_edit_modal.hbs

```diff
@@ -2,29 +2,48 @@
-            Shelf Thresholds:
-          <div class="form-group">
-            <label for="modal_shelf_minimum_threshold" class="col-form-label">Minimum:</label>
-            <input type="number" class="form-control" min='0'id="modal_shelf_minimum_threshold" name="modal_shelf_minimum_threshold">
-          </div>
-          <div class="form-group">
-            <label for="modal_shelf_maximum_threshold" class="col-form-label">Maximum:</label>
-            <input type="number" class="form-control" min='0' id="modal_shelf_maximum_threshold" name="modal_shelf_maximum_threshold">
-            <div class="invalid-feedback">
-              Shelf maximum must be more than or equal to shelf minimum.
-            </div>
-          </div>
-          <div class="form-group">
-            <input type="hidden" class="form-control" id="modal_shelf_id" name="modal_shelf_id">
-          </div>
-        </form>
+          <form class="needs-validation" action="/inventory/modify_item" method="POST" id="modal_shelf">
+              Shelf Thresholds:
+              <div class="form-group">
+                  <label for="modal_shelf_minimum_threshold" class="col-form-label">Minimum:</label>
+                  <input type="number" class="form-control" min='0' id="modal_shelf_minimum_threshold" name="modal_shelf_minimum_threshold">
+              </div>
+              <div class="form-group">
+                  <label for="modal_shelf_maximum_threshold" class="col-form-label">Maximum:</label>
+                  <input type="number" class="form-control" min='0' id="modal_shelf_maximum_threshold" name="modal_shelf_maximum_threshold">
+                  <div class="invalid-feedback">
+                      Shelf maximum must be more than or equal to shelf minimum.
+                  </div>
+              </div>
+              <div class="form-group">
+                  <input type="hidden" class="form-control" id="modal_shelf_id" name="modal_shelf_id">
+              </div>
+
+
+              Warehouse Thresholds:
+              <div class="form-group">
+                  <label for="modal_wh_minimum_threshold" class="col-form-label">Minimum:</label>
+                  <input type="number" class="form-control" min='0' id="modal_wh_minimum_threshold" name="modal_wh_minimum_threshold">
+              </div>
+              <div class="form-group">
+                  <label for="modal_wh_maximum_threshold" class="col-form-label">Maximum:</label>
+                  <input type="number" class="form-control" min='0' id="modal_wh_maximum_threshold" name="modal_wh_maximum_threshold">
+                  <div class="invalid-feedback">
+                      Warehouse maximum must be more than or equal to warehouse minimum.
+                  </div>
+              </div>
+              <div class="form-group">
+                  <input type="hidden" class="form-control" id="modal_shelf_id" name="modal_shelf_id">

```

Comment:
To be honest, this duplicate, there is only one ID per product so his code could use gt_shelf_id. We could easily refactor and just call gt_product_id as well.

Reply:
@gkochera @quinn-codes-synthesis ah okay! Ya, `gt_product_id` would be a little more clear to someone less familiar like me :)